psamm/tests/test_yaml.py

```diff
@@ -114,6 +114,97 @@ def write_model_file(self, filename, contents):
             f.write(contents)
         return path
 
+    def test_long_string_model(self):
+        longString = '''---
+            name: Test model
+            biomass: rxn_1
+            reactions:
+              - id: rxn_1
+                equation: '|A_\u2206[e]| => |B[c]|'
+                genes:
+                  - gene_1
+                  - gene_2
+              - id: rxn_2_\u03c0
+                equation: '|B[c]| => |C[e]|'
+                genes: 'gene_3 or (gene_4 and gene_5)'
+            compounds:
+              - id: A_\u2206
+              - id: B
+              - id: C
+            media:
+              - compartment: e
+                compounds:
+                  - id: A_\u2206
+                  - id: C
+            limits:
+              - reaction: rxn_2_\u03c0
+                upper: 100
+            '''
+        m = native.NativeModel(longString)
+        self.assertEqual(m.get_name(), 'Test model')
+        self.assertEqual(m.get_biomass_reaction(), 'rxn_1')
+        self.assertEqual(m.get_extracellular_compartment(), 'e')
+
+    def test_long_string_model_include(self):
+        longString = '''---
+            name: Test model
+            biomass: rxn_1
+            reactions:
+              - id: rxn_1
+                equation: '|A_\u2206[e]| => |B[c]|'
+                genes:
+                  - gene_1
+                  - gene_2
+              - id: rxn_2_\u03c0
+                equation: '|B[c]| => |C[e]|'
+                genes: 'gene_3 or (gene_4 and gene_5)'
+            compounds:
+              - id: A_\u2206
+              - id: B
+              - id: C
+            media:
+              - include: medium.yaml
+            limits:
+              - reaction: rxn_2_\u03c0
+                upper: 100
+            '''
```

Comment:
Maybe this model could be more minimal? For example only include `name`, `biomass` and `reactions` and then put the `include` in the `reactions` list. The test would be on `parse_reactions()` instead. This is not strictly necessary but I think it would make the test easier to understand.